scripts/make-serializer-pickle-files

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+cat << "EOF" > serializer-pickle-files.lock
+# This file was automatically generated by running `make serializer-pickle-files.lock`.
+# The purpose of this file is to prevent people from innocently adding new usages
+# of serializer='pickle' in celery @task declarations, since such usage is considered deprecated in this project.
+# As we remove usages, this file will be updated to reflect that and prevent unintended backsliding.
+
+EOF
+
+grep -R "serializer='pickle'" --include='*.py' . | cut -d':' -f1 | cut -c3- | sort | uniq -c | awk '{printf "%s\t%s\n", $1, $2}' >> serializer-pickle-files.lock
```

Comment:
How about string formatting here to enforce right-aligned space (` `) padding instead?  This guarantees consistent whitespace (doesn't depend on GNU vs BSD `uniq -c`, who's output formats differ).  Tab spacing isn't guaranteed to line up because tab width can differ from system-to-system. (You can bump it to `%4s` if you're worried about more than 999 matches in a file :wink:) 
```suggestion
grep -R "serializer='pickle'" --include='*.py' . | cut -d':' -f1 | cut -c3- | sort | uniq -c | awk '{printf "%3s %s\n", $1, $2}' >> serializer-pickle-files.lock
```

Reply:
Also, is it important that `grep` follow symlinks (`-R` vs `-r`)?  I have symlinks inside a git-ignored directory in my repo for docker purposes, and this script emits filesystem permission warnings when I run it with `-R` (because some of the contents deep in those link targets are root-owned). I'm unaware of symlinks in the repo that this script would need to follow.

Reply:
Ah, I think this `-R/-r` distinction may also be different on mac, but `-r` works for both, so I'll change it to that.

I'm pretty sure the `\t` thing works just fine because it results in an actual tab character in the file (not a variable number of ` ` characters). No?

Reply:
I took a closer look at what you'd suggested above, and I'm happy to go with that (but with the `-r`). Thanks for the contribution!

Reply:
> works just fine because it results in an actual tab character in the file

Correct, but some systems/applications will _display_ that as taking up 8 character positions, while others 4 (this is often further customizable by the user, e.g. VS Code, Sublime Text, PyCharm, Eclipse, Vim, etc). This means that what "lines up" on your system may not line up on another, and in many cases, the user can't change their tab width (Notepad, Web browsers, etc) or they may not know how (various terminal emulators, etc).